### ISSUE TITLE
token support rather than using user/pw would be preferrable

### ISSUE THREAD

**Issue by cglewis** [neyric/gh-issues-gantt#18]

**Comment by maximerety** [comment]
Good point @cglewis.

As it's a bit of work to add support for OAuth2 tokens, I don't think anybody will volunteer to develop this feature any time soon.

If this feature is a requirement for your use case, you're more than welcome to submit a pull request!

Cheers,

**Comment by srp** [comment]
I found that I was able to use my token in the password field and that worked.

**Comment by ijstokes** [comment]
Yes, just put your token in as your password and that works.  Does require that you share your username.
